packages/itwinui-css/src/utils/notification-marker/notification-marker.scss

```diff
@@ -2,25 +2,38 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 
 $iui-notification-marker-border-width: var(--iui-size-3xs);
+$iui-notification-before-size: calc(var(--iui-size-xs) + $iui-notification-marker-border-width * 2);
+$iui-notification-before-size-radius: calc($iui-notification-before-size * 0.5);
 
 @mixin iui-notification-marker($status: primary) {
   position: relative;
 
   &::before {
     content: '';
     position: absolute;
-    width: calc(var(--iui-size-xs) + $iui-notification-marker-border-width * 2);
-    height: calc(var(--iui-size-xs) + $iui-notification-marker-border-width * 2);
-    top: calc(var(--iui-size-s) * -0.5);
-    right: calc(var(--iui-size-s) * -0.5);
+    width: calc($iui-notification-before-size - 2 * $iui-notification-marker-border-width);
+    height: calc($iui-notification-before-size - 2 * $iui-notification-marker-border-width);
+    top: calc($iui-notification-before-size-radius + var(--iui-size-s) * -0.5);
+    right: calc($iui-notification-before-size-radius + var(--iui-size-s) * -0.5);
+    transform: translate(50%, -50%);
+
     border-radius: 100%;
     background-color: var(--iui-color-foreground-#{$status});
-    border: $iui-notification-marker-border-width solid var(--iui-color-background-1);
     @media (prefers-reduced-motion: no-preference) {
       transition: background-color var(--iui-duration-1) ease-out, border-color var(--iui-duration-1) ease-out;
     }
   }
 
+  > * {
+    mask: radial-gradient(
+      ellipse $iui-notification-before-size-radius $iui-notification-before-size-radius at
+        calc(100% - $iui-notification-before-size-radius - var(--iui-size-s) * -0.5)
+        calc($iui-notification-before-size-radius + var(--iui-size-s) * -0.5),
+      transparent 95%,
+      #000 100%
```

Comment:
nit: why not `black`?

Reply:
I was getting this error. Maybe because of the linting restrictions for named colors? [(Stackoverflow)](https://stackoverflow.com/a/50484451/11547064)
![image](https://user-images.githubusercontent.com/45748283/192552243-5e50dc31-bc5b-487a-b027-15df67956e91.png)